From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@ClarityQueen__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (v-diree). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

NingKuang committed:
fixing the URL for OMS views

NingKuang committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into CF-working-doc

NingKuang committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into CF-working-doc

updating local branch.

NingKuang committed:
Fix a typo.

rachelappel committed:
delete files and update toc

rachelappel committed:
fix links

rachelappel committed:
fix more links

NingKuang committed:
update with feedback

NingKuang committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into CF-working-doc
updating with feedback

NingKuang committed:
remove the template reference

NingKuang committed:
updating the formate

NingKuang committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into CF-working-doc
Updating format and removing instructions

NingKuang committed:
updating the format

NingKuang committed:
updating the format

NingKuang committed:
updating the format

NingKuang committed:
updating the format

NingKuang committed:
updating the format

NingKuang committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into CF-working-doc
updating format

NingKuang committed:
updating the format

NingKuang committed:
final updates

NingKuang committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into CF-working-doc
final updates

NingKuang committed:
final updates2

NingKuang committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into CF-working-doc
update to address tool feedback

rachelappel committed:
Use Custom Docker image on Web App on Linux

rachelappel committed:
added images

rachelappel committed:
readd mods

rachelappel committed:
private reg update

rachelappel committed:
Node on Linux MVC

rachelappel committed:
fix link

rachelappel committed:
ssh updates

rachelappel committed:
more cleanup

rachelappel committed:
changed field markers;some cleanup

rachelappel committed:
changed h2

rachelappel committed:
changed link

rachelappel committed:
minor

rachelappel committed:
changes

rachelappel committed:
modifications after feedback

rachelappel committed:
modifications per feedback

rachelappel committed:
post feedback mods, 8.1 update

adegeo committed:
Updated per feedback

v-anpasi committed:
Merge pull request #20357 from v-anpasi/release-web-apps-containers

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Merge pull request #20520 from v-anpasi/release-web-apps-containers

cephalin committed:
add /containers folder

cephalin committed:
remove dup

cephalin committed:
add toc.md

alexkarcher-msft committed:
Fixing capitalization

cephalin committed:
debugging

cephalin committed:
validation

cephalin committed:
validation

cephalin committed:
validation

cephalin committed:
validation

paulathurman committed:
edit pass: site-recovery-dynamicsax

cephalin committed:
add bread crumbs

cephalin committed:
debug

v-anpasi committed:
Bringing even with master.

cephalin committed:
debug

cephalin committed:
debug

cephalin committed:
debug

cephalin committed:
debug

cephalin committed:
debug

cephalin committed:
debug

cephalin committed:
debug

cephalin committed:
debug

v-anpasi committed:
Merge pull request #20683 from v-anpasi/release-web-apps-containers

cephalin committed:
quickstarts

paulathurman committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into site-recovery-dynamicsax

paulathurman committed:
edit pass: site-recovery-dynamicsax

paulathurman committed:
edit pass: site-recovery-articles

paulathurman committed:
edit pass: site-recovery-article

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Merge pull request #20828 from v-anpasi/release-web-apps-containers

cephalin committed:
quickstarts

cephalin committed:
big

rachelappel committed:
minor changes

rachelappel committed:
clean up link

v-anpasi committed:
Bringing even with master.

rachelappel committed:
updates

v-anpasi committed:
Merge pull request #20993 from v-anpasi/release-web-apps-containers

cephalin committed:
debug

cephalin committed:
merge conflict

cephalin committed:
validation

cephalin committed:
node.js tutorial

cephalin committed:
validation

cephalin committed:
validation

v-anpasi committed:
Bringing even with master.

v-anpasi committed:
Merge pull request #21156 from v-anpasi/release-web-apps-containers

NingKuang committed:
update based on Iain's feedback

NingKuang committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into CF-working-doc
update based on Iain's feedback

rachelappel committed:
update for merge

CarlRabeler committed:
update

CarlRabeler committed:
checkin

CarlRabeler committed:
syncx

v-anpasi committed:
Bringing even with master.

CarlRabeler committed:
Merge branch 'perf-tut2' of https://github.com/CarlRabeler/azure-docs-pr into migrate-tutorial

v-anpasi committed:
Merge pull request #21299 from v-anpasi/release-web-apps-containers

Commit:
Merge branch 'docker-webapp' of github.com:rachelappel/azure-docs-pr into cfowler-web-app-containers

Commit:
Merge branch 'node-2' of github.com:rachelappel/azure-docs-pr into cfowler-web-app-containers

Commit:
Merge branch 'ssh-include-del' of github.com:rachelappel/azure-docs-pr into cfowler-web-app-containers

Commit:
removing preview include to trigger warnings in build

Commit:
Removing old Node.js topic, replaced with new topic

Commit:
updating custom docker image tutorial

Commit:
Merge branch 'app-service-linux-deployment' of https://github.com/nickwalkmsft/azure-docs-pr into cfowler-web-app-containers

Commit:
removing the preview messages

Commit:
redirecting removed topics, removing other broken references

Commit:
removing preview message

NingKuang committed:
adding template link

NingKuang committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into cf-working-doc
adding template link

lastcoolnameleft committed:
Update app-service-linux-faq.md

After spending time debugging why my dependencies weren't loading, I discovered that the app uses composer.lock NOT composer.json to include new packages.

NingKuang committed:
adjusting title

NingKuang committed:
adjusting new links

NingKuang committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into cf-working-doc
fixing the name length

NingKuang committed:
removing dup link

cephalin committed:
core & php tutorials

cephalin committed:
merge from master

cephalin committed:
merge conflict

cephalin committed:
fixed links

NingKuang committed:
removing keywords

cephalin committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into release-web-apps-containers

cephalin committed:
redirects

cephalin committed:
validation

cephalin committed:
broken links

cephalin committed:
broken links

cephalin committed:
custom image quickstart

cephalin committed:
broken links

cephalin committed:
validation

cephalin committed:
validation

cephalin committed:
validation

alexkarcher-msft committed:
Adding instructions for anymous proxy

rmca14 committed:
Merge pull request #20542 from cephalin/release-web-apps-containers

add /containers folder

alexkarcher-msft committed:
Merge remote-tracking branch 'Source/master'

# Conflicts:
#	articles/azure-functions/functions-api-definition-getting-started.md

alexkarcher-msft committed:
Making backend/frontend distinction more clear

Commit:
updated for Web App for Containers

Commit:
adding Ruby updates

Commit:
Updates to the Linux ToC

Commit:
merging directory structure changes

Commit:
removing duplicates

Commit:
fixing file linking issues

Commit:
removing duplicates

Commit:
rename to match convention

Commit:
changed images

Commit:
formatting updates

nickwalkmsft committed:
resolve conflict

Commit:
add Web Apps for Containers Landing page

Commit:
removing Ruby tutorial from ToC

Commit:
removed empty ToC items

Commit:
yaml formatting issue

Commit:
update metadata & made steps optional

Commit:
updated to include optional in inline archors

alexkarcher-msft committed:
Removing file and acrolinxing

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into class0906

NingKuang committed:
move file under /cloudfoundry

NingKuang committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into cf-working-doc
moving the file to cloudfoundry folder, addressing editoring feedback

Commit:
addressing blocking feedback

cephalin committed:
Update quickstart-dotnetcore.md

cephalin committed:
Update toc.yml

cephalin committed:
Update tutorial-dotnetcore-sqldb-app.md

cephalin committed:
Update tutorial-dotnetcore-sqldb-app.md

PRMerger17 committed:
Merge pull request #21433 from cephalin/patch-4

Update quickstart-dotnetcore.md

PRMerger18 committed:
Merge pull request #21471 from cephalin/patch-7

Update tutorial-dotnetcore-sqldb-app.md

cephalin committed:
reviewer feedback

ktoliver committed:
Merge pull request #21434 from cephalin/patch-6

Update toc.yml

cephalin committed:
Merge branch 'release-web-apps-containers' of https://github.com/MicrosoftDocs/azure-docs-pr into release-web-apps-containers

cephalin committed:
validation

cephalin committed:
reviewer feedback

ktoliver committed:
Merge pull request #21493 from cephalin/release-web-apps-containers

Release web apps containers

Commit:
Branding updates

Commit:
globalizing URL

Commit:
updates to capitalization in ToC

ktoliver committed:
Merge pull request #21303 from SyntaxC4-MSFT/cfowler-web-app-containers

Adding Web App on Linux/for Containers topics

Commit:
fixing merge conflicts

CarlRabeler committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into migrate-tutorial

rmca14 committed:
Merge pull request #21534 from SyntaxC4-MSFT/master

Fixing Merge Conflicts

curtand committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into class0906

nitinme committed:
Adding a link to the new Dev Guide

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-toc-landing-updates

alexkarcher-msft committed:
Changing list content to numbered list

alexkarcher-msft committed:
Adding missing colons

alexkarcher-msft committed:
Misc numbering formatting fixes

nitinme committed:
Merge remote-tracking branch 'refs/remotes/MicrosoftDocs/master' into nitinme-toc-landing-updates

Commit:
updating changed filenames in redirects file

rmca14 committed:
Merge pull request #21584 from SyntaxC4-MSFT/master

Fixing redirects

rmca14 committed:
Merge pull request #21355 from MicrosoftDocs/release-web-apps-containers

Release web apps containers

alexkarcher-msft committed:
Fixing table formatting

curtand committed:
small update for comprehension

curtand committed:
PM name change

alexkarcher-msft committed:
Fixing note rendering issue

curtand committed:
updated resource access article

v-thepet committed:
Merge pull request #21356 from alexkarcher-msft/master

Updating Proxies Instructions

v-thepet committed:
Merge pull request #21582 from nitinme/nitinme-toc-landing-updates

Landing page update for HDI Dev Guide

tfitzmac committed:
adding event grid data migration

Commit:
Merge pull request #21591 from curtand/class0906

small update for comprehension

robinsh committed:
Update AzCopy article, fix branding, add section req by @YoungjaeKim

v-thepet committed:
Update app-service-linux-choose-deployment-type.md

v-thepet committed:
Delete app-service-linux-choose-deployment-type.md

ShawnJackson committed:
Update site-recovery-dynamicsax.md

robinsh committed:
Update storage explorers

ShawnJackson committed:
Update site-recovery-vmware-to-azure-push-install-error-codes.md

barbkess committed:
TOC updates

ShawnJackson committed:
Merge pull request #20740 from paulathurman/site-recovery-dynamicsax

edit pass: Site recovery dynamicsax

CarlRabeler committed:
Update sql-database-auditing-and-threat-detection-powershell.md

robinsh committed:
Make titles consistent (azcopy)

CarlRabeler committed:
migrate

Commit:
ca03

PRMerger14 committed:
Merge pull request #21607 from CarlRabeler/patch-333

Update sql-database-auditing-and-threat-detection-powershell.md

CarlRabeler committed:
redirection

CarlRabeler committed:
more

robinsh committed:
Apply public PR changes to I/E article.

CarlRabeler committed:
png

Commit:
ca03

CarlRabeler committed:
acro

CarlRabeler committed:
acro

CarlRabeler committed:
dup

Commit:
ca03

robinsh committed:
Try to raise the acrolinx score for azcopy

PRMerger19 committed:
Merge pull request #21618 from MarkusVi/ca03

ca03

GitHubber17 committed:
Merge pull request #21601 from MicrosoftDocs/v-thepet-patch-2-1

production bug: Update app-service-linux-choose-deployment-type.md

NingKuang committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into cf-working-doc
remove the file from virtual-machine/linux folder

NingKuang committed:
removing the flie under virtual-machine/linux, fixing a format issue

Lisaco88 committed:
Merge pull request #21598 from tfitzmac/0906hubs

adding event grid data migration

GitHubber17 committed:
Merge pull request #21599 from robinsh/robinsh-handle-public-prs

Update AzCopy article, fix branding

GitHubber17 committed:
Merge pull request #21603 from barbkess/resources

TOC updates

Juliako committed:
Update media-services-rest-get-started.md

PRMerger16 committed:
Merge pull request #21624 from Juliako/patch-15

Update media-services-rest-get-started.md

NingKuang committed:
Revert "removing the flie under virtual-machine/linux, fixing a format issue"

This reverts commit cebb26303b35d395fbcb3865fa91725ccd9b5abf.

revert the change as it fails

NingKuang committed:
remove the file under virtual-machine folder

NingKuang committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into cf-working-doc
remove file under virtual-machine linux

pvrk committed:
Update backup-try-azure-backup-in-10-mins.md

Fixing a typo from ne to net

Commit:
Merge pull request #21701 from pvrk/patch-14

Update backup-try-azure-backup-in-10-mins.md

YuriDio committed:
updating articles

YuriDio committed:
updating articles 2

Commit:
updates for freshness

Commit:
formatting

CarlRabeler committed:
Update sql-database-import.md

PRMerger13 committed:
Merge pull request #21711 from Blackmist/fresh5

updates for freshness

mimig1 committed:
Cosmos DB: Added Try Cosmos DB subscription info.

mimig1 committed:
Updated free account faq section

mimig1 committed:
Cosmos DB: Added tip #3.

PRMerger20 committed:
Merge pull request #21713 from CarlRabeler/patch-334

Update sql-database-import.md

CarlRabeler committed:
edits

Commit:
Merge pull request #21715 from mimig1/09072017lsimona

Cosmos DB: Added tip #3.

rmca14 committed:
Merge pull request #21708 from YuriDio/AugustUpdates_8

updating articles

tfitzmac committed:
wording changes

rjagiewich committed:
Merge pull request #21221 from CarlRabeler/migrate-tutorial

update

PRMerger14 committed:
Merge pull request #21720 from tfitzmac/0907hubgrid

wording changes

PRMerger16 committed:
Merge pull request #21714 from mimig1/09072017l

Cosmos DB: Added Try Cosmos DB subscription info.

rjagiewich committed:
Merge pull request #20243 from Thraka/svc-new

Updated per feedback

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

rjagiewich committed:
Merge pull request #21429 from NingKuang/cf-working-doc

Cf working doc

v-alje committed:
Merge pull request #21726 from MicrosoftDocs/FromPublicRepo

From public repo
